mifosng-android/src/release/java/com/mifos/api/local/MifosSQLCipherHelper.java

```diff
@@ -0,0 +1,18 @@
+package com.mifos.api.local;
+
+import com.raizlabs.android.dbflow.config.DatabaseDefinition;
+import com.raizlabs.android.dbflow.structure.database.DatabaseHelperListener;
+import com.raizlabs.dbflow.android.sqlcipher.SQLCipherOpenHelper;
+
+public class MifosSQLCipherHelper extends SQLCipherOpenHelper {
+
+    public MifosSQLCipherHelper(DatabaseDefinition databaseDefinition,
+                                DatabaseHelperListener listener) {
+        super(databaseDefinition, listener);
+    }
+
+    @Override
+    protected String getCipherSecret() {
+        return "dbflow-rules";
```

Comment:
This line of code is the problem, Secret is something that people cannot know. We need to find a way to make it secure.